templates/ironicconductor/bin/init.sh

```diff
@@ -24,6 +24,7 @@ export PODINDEX=$(echo ${HOSTNAME##*-})
 IRONICPASSWORD=${IronicPassword:-""}
 TRANSPORTURL=${TransportURL:-""}
 QUORUMQUEUES=${QuorumQueues:-"false"}
+GRACEFUL_SHUTDOWN_TIMEOUT=${GracefulShutdownTimeout:-""}
```

Comment:
These init.sh changes can be removed from the commit too now that it is happening in 01-conductor.conf